build.gradle

```diff
@@ -15,26 +15,47 @@
  */
 
 task wrapper(type: Wrapper) {
-  gradleVersion = '1.11'
+    gradleVersion = '2.2.1'
 }
 
 buildscript {
-  apply from: "https://github.com/rosjava/rosjava_bootstrap/raw/indigo/buildscript.gradle"
+    apply from: "https://github.com/talregev/rosjava_mvn_repo/raw/master/buildscript.gradle"
+    dependencies {
+        classpath "com.android.tools.build:gradle:1.0.1"
+    }
 }
 
 apply plugin: 'catkin'
 
 allprojects {
+    apply from: "https://github.com/talregev/rosjava_mvn_repo/raw/master/maven.gradle"
```

Comment:
Might do this for the non-api parts of the ros gradle plugin. It's an extra line, but that visibility will help people contribute when changes happen.